.changeset/cyan-ants-exercise.md

```diff
@@ -0,0 +1,8 @@
+---
+"@marigold/docs": patch
+"@marigold/components": minor
+"@marigold/theme-b2b": patch
+"@marigold/theme-core": patch
+---
+
+RAC: Menu react aria components
```

Comment:
This breaking change info should also be added here: https://github.com/marigold-ui/marigold/pull/3546/files#diff-7617ce6947d6789d501bb7df466ddca07d7d0aa5264a4c3f43f9cdbbed05f22b

Reply:
done